vultr.php

```diff
@@ -2221,6 +2221,13 @@ private function getTabActions($package, $service, array $post = null, $client =
                             }
                         }
                         break;
+                    case 'enable_ipv6':
```

Comment:
How come this is an action that must manually be set on the service instead of being set automatically when the service is created?

This option also looks like it is a one-time use action. You can enable it if it's disabled, but that's it. Is there a way to disable the option if it's enabled? Why is it shown on the actions page if there's nothing else we can do with it?

Reply:
I wrote this a little while ago but I think it was a problem with the server not being responsive to the request immediately after creation.

I believe you are correct about it being one-time and no I don't think you can disable after enabled.  It is on the actions page because I don't know of another place to put it.  Right now I disable it if already enabled, but if you like we could hide it entirely.

Reply:
Retested and I can confirm that making the call right after server creation fails because it is still spinning up